cmd/catalyst-node/catalyst-node.go

```diff
@@ -452,7 +461,8 @@ func parsePlaybackID(path string) (string, bool) {
 	if len(m) < 2 {
 		return "", false
 	}
-	return m[1], true
+	slice := strings.Split(m[1], "+")
```

Comment:
I don't understand this part. Why do we split by `+`? Maybe add a comment with the explanation.

Reply:
Added a comment explaining this.